cfn-resources/datalakes/cmd/resource/resource.go

```diff
@@ -249,20 +237,20 @@ func List(req handler.Request, prevModel *Model, currentModel *Model) (handler.P
 		ResourceModels:  models,
 	}, nil
 }
-func flattenAWSBlock(awsConfig *mongodbatlas.CloudProviderConfig) *DataLakeAWSCloudProviderConfigView {
+func flattenAWSBlock(awsConfig *admin.DataLakeCloudProviderConfig) *DataLakeAWSCloudProviderConfigView {
 	if awsConfig == nil {
```

Comment:
can aws.AWSConfig also be nil?     and &awsConfig.AWSConfig.IAMUserARN, etc. ?

Reply:
no, `AWSConfig` is required